docs/manage/data/query.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Query Data"
```

Comment:
Similar to my other comment, worth pointing out that this is specific to more flexible querying via SQL/MQL, whereas you can already query all data with a Filter (links in other comment)